### DISCUSSION TITLE
How to install Podman 4.6 in Ubuntu?

### DISCUSSION THREAD

**Discussion by ashishsinghin** [containers/podman#19672] [General]
I install podman using apt-get package manager but it install version 3.4.4 that is older one.
How to install latest version of podman in ubuntu any guide?

**Comment by afbjorklund** [comment]
For history, see:

* https://github.com/containers/podman/discussions/17362

* https://packages.ubuntu.com/search?keywords=podman&searchon=names&exact=1&suite=all&section=all

There are details on how to install the "unstable" from the Kubic repo: https://podman.io/docs/installation#ubuntu

You can also use Podman Machine to install a Fedora VM, but it is not supported (on Linux) by Podman Desktop.
